src/store.ts

```diff
@@ -70,6 +70,9 @@ export interface SDKConfig {
     v1SecureServiceUrl?: string;
     v2SecureServiceUrl?: string;
     v3SecureServiceUrl?: string;
+
+    wrapperSDKName: string;
```

Comment:
Looks like there's a build test that is failing because this field is missing in the interface for SDKInitConfig in `sdkRuntimeModels.ts`